bearer/bearer.go

```diff
@@ -244,16 +244,16 @@ func (b Token) AssertUser(id user.ID) bool {
 	return !b.targetUserSet || b.targetUser.Equals(id)
 }
 
-// Sign calculates and writes signature of the Token data using issuer's signer.
+// Sign calculates and writes signature of the [Token] data using issuer's signer.
 // Returns signature calculation errors.
 //
-// Sign MUST be called if Token is going to be transmitted over
+// Sign MUST be called if [Token] is going to be transmitted over
 // NeoFS API V2 protocol.
 //
-// Note that any Token mutation is likely to break the signature, so it is
+// Note that any [Token] mutation is likely to break the signature, so it is
```

Comment:
These are not related, of course. I meant only those lines that were changed. Can we separate these changes into a different commit?

Reply:
We can, but why not related? Updating documentation in the same place as `See also [Token.VerifySignature], [Token.Issuer], [Token.SignedData].` What the difference between this row and other documentation in this function?

Reply:
You're inevitably touching "See also [Token.VerifySignature], [Token.Issuer], [Token.SignedData]" in your PR, so you can change links as well. You don't need to touch these comments to deal with `SignedData`, that's why it's a separate change. Good one! But separate.

Reply:
OK, I see, I will separate these changes

Reply:
Moved to separate commit